.github/workflows/ci_action.yml

```diff
@@ -0,0 +1,31 @@
+name: build
+
+on:
+  push:
+
+jobs:
+
+  check-code-black-isort:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout branch
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ env.CHECKOUT_BRANCH }}
+    
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.8"
```

Comment:
This part might be problematic. Some of the examples we have, could potentially only work on some old versions (of sh-py/eo-learn/Python in general). We've been discussing that perhaps we'd rather require that each example comes with specs for env).

Reply:
The good part about `black` and `isort` is that they don't require any kind of package installation (apart from themselves). Therefore it shouldn't really matter what python version the formatters are installed in. Until we beef it up (unlikely to happen soon) I don't think it should be much of an issue.

Reply:
Good. Then the PR has my blessing ;)